ember-bootstrap/addon/template-registry.ts

```diff
@@ -1,9 +1,11 @@
 import type BsButton from './components/bs-button';
 import type BsCollapse from './components/bs-collapse';
 import type BsSpinner from './components/bs-spinner';
+import type BsProgressBar from 'ember-bootstrap/components/bs-progress/bar';
 
 export default interface EmberBootstrapRegistry {
   BsButton: typeof BsButton;
   BsCollapse: typeof BsCollapse;
   BsSpinner: typeof BsSpinner;
+  'bs-progress/bar': typeof BsProgressBar;
```

Comment:
I think this should have `BsProgress` (not dasherized), and only that. The bar component is yielded, but should not be invoked directly (IIRC)

Reply:
Ah, these should've been added to the internal types I think.
We still use the dasherized version with the `component` helper. Once everything is in GJS/GTS we should be able to get rid of those.